index.html

```diff
@@ -101,127 +94,136 @@ <h2 class="header-text">My Recent Works</h2>
-                </p>
-                <div>
-                    <ul class="card-tags">
-                        <li><button class="tag blur">html</button></li>
-                        <li><button class="tag blur">bootstrap</button></li>
-                        <li><button class="tag blur">Ruby</button></li>
-                    </ul>
+            <div class="work-container">
+                <h2 class="work-header">Multi-Post Stories</h2>
+                <div class="work-description">
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
                 </div>
-                <button class="btn-max-width">See Project</button>
-            </div>
-            <!-- second work card -->
-            <div class="work-card">
-                <h2 class="card-header">Professional Art <br> Printing Data</h2>
-                <p class="card-description">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-                </p>
                 <div>
-                    <ul class="card-tags">
-                        <li><button class="tag blur">html</button></li>
-                        <li><button class="tag blur">bootstrap</button></li>
-                        <li><button class="tag blur">Ruby</button></li>
+                    <ul class="tags-container">
+                        <li>
+                            <button class="tag">CSS</button>
+                        </li>
+                        <li><button class="tag">html</button></li>
+                        <li><button class="tag">bootstrap</button></li>
+                        <li><button class="tag">Ruby</button></li>
                     </ul>
                 </div>
-                <button class="btn-max-width">See Project</button>
+                <button class="main-btn">See Project</button>
             </div>
-             <!-- third -->
-            <div class="work-card">
-                <h2 class="card-header">Professional Art <br> Printing Data</h2>
-                <p class="card-description">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-                </p>
-                <div>
-                    <ul class="card-tags">
-                        <li><button class="tag blur">html</button></li>
-                        <li><button class="tag blur">bootstrap</button></li>
-                        <li><button class="tag blur">Ruby</button></li>
-                    </ul>
+        </div>
+        <!-- work Card -->
+        <div class="work-cards-container">
```

Comment:
- The gap between `main-work-section` and `work-cards-container` is more than the specified gap in the Figma template. Kindly fix the issue, please. 💯 
- Moreover, the recent work should be aligned on the center of the page as per the Figma template. Please can you fix this issue as well to make your UI better? 😃 

Figma Design
<img width="894" alt="Screenshot 2022-12-10 at 10 22 37 PM" src="https://user-images.githubusercontent.com/24832846/206867404-57b24539-f878-4e07-9f14-8bec35bbc0f6.png">

Your Design
<img width="1439" alt="Screenshot 2022-12-10 at 10 22 56 PM" src="https://user-images.githubusercontent.com/24832846/206867401-200e9e41-7ae1-4d08-b5d7-19a9f6a20d92.png">

Reply:
- I have updated the margin and padding of elements to match the Figma design
- I have added a hover effect to change the style of the cards.